pkg/skr/cloudresources/deleteUiCms.go

```diff
@@ -0,0 +1,43 @@
+package cloudresources
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+	"github.com/kyma-project/cloud-manager/pkg/util"
+)
+
+func deleteUiCms(ctx context.Context, st composed.State) (error, context.Context) {
+
+	state := st.(*State)
+	logger := composed.LoggerFromCtx(ctx)
+
+	uiCmList := util.NewUiCmListUnstructured()
+	err := state.Cluster().ApiReader().List(ctx, uiCmList)
```

Comment:
Scope the list to single namespace where Busola extension are installed (kyma-system I guess), and filter by labels at the same time:

```suggestion
	err := state.Cluster().ApiReader().List(
		ctx,
		list,
		client.MatchingLabels{
			"busola.io/extension": "resource",
		},
		client.InNamespace("kyma-system"),
	)
```